### PR TITLE
Update README as expected

### PR DESCRIPTION
**Pull request by damien-rivet**
# Title

It was completely necessary!

### CODE REVIEW

File: README.md
Comment (damien-rivet) [inline README.md:1]: This file is the most perfect one!

File: README.md [hunk 1]
```diff
@@ -2,3 +2,7 @@
 Ce repo contient le code de base pour l'exercice P9 que vous trouverez dans le parcours OpenClassrooms : Développeur d'application - iOS.
 
 Le projet est en Swift et utilise le framework UIKit.
+
+# Extras
```
Comment (damien-rivet) [inline README.md:6]: I would not call this section like that:

```suggestion
# Extraneous
```

File: README.md [hunk 2]
```diff
@@ -2,3 +2,7 @@
 Ce repo contient le code de base pour l'exercice P9 que vous trouverez dans le parcours OpenClassrooms : Développeur d'application - iOS.
 
 Le projet est en Swift et utilise le framework UIKit.
+
```
Comment (damien-rivet) [inline README.md:5]: IT WORKS

File: README.md [hunk 3]
```diff
@@ -2,3 +2,7 @@
...
+# Extras
+
+It works so well!
```
Comment (damien-rivet) [inline README.md:8]: OH NOES

**Review by damien-rivet — COMMENTED** [review]
IT WORKS SO WELL

File: README.md (hunk 2 above)
Comment (damien-rivet) [reply README.md:5]: No it does not.

**Review by MargotPasquali — APPROVED** [review]
ok perfect

**Pull request closed without merging**